src/playground/README

```diff
@@ -0,0 +1,34 @@
+## How to install
```

Comment:
Maybe the main readme, can just link here?

Reply:
I figured out that this README would be the only readme file in subdirectories, so I am removing it, same as I removed package.json and LICENSE. Let's keep things in one place if we have a monorepo.  
All the contents are in the main README.md anyway.